eZ/Publish/Core/Persistence/Cache/Tests/Tags/TagGeneratorTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\Persistence\Cache\Tests\Tags;
```

Comment:
-> `tests/lib/Persistence/Cache/Tag`
```suggestion
namespace Ibexa\Tests\Core\Persistence\Cache\Tag;
```